front/src/reducers/mapViewer/index.ts

```diff
@@ -0,0 +1,24 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+import type { InfraStateType } from 'reducers/infra';
+import { infraState, buildInfraStateReducers } from 'reducers/infra';
+
+export interface MapViewerStateType extends InfraStateType {}
+
+export const mapViewerInitialState: MapViewerStateType = {
+  ...infraState,
+};
+
+export const mapViewerSlice = createSlice({
+  name: 'mapViewer',
+  initialState: mapViewerInitialState,
+  reducers: {
+    ...buildInfraStateReducers<MapViewerStateType>(),
+  },
+});
+
+export const mapViewerSliceActions = mapViewerSlice.actions;
+
+export type mapViewerSliceType = typeof mapViewerSlice;
```

Comment:
Pascal case + remove Type
```suggestion
export type MapViewerSlice = typeof mapViewerSlice;
```